fastify.js

```diff
@@ -548,13 +548,13 @@ function build (options) {
       }
     }
 
-    options = Object.assign({}, options, {
+    const opt = Object.assign({}, options, {
       method,
       url,
       handler: handler || (options && options.handler)
     })
-
-    return _fastify.route(options)
+    options.config && (opt.config = JSON.parse(JSON.stringify(options.config)))
```

Comment:
Can you please use https://github.com/davidmarkclements/rfdc instead? Functions are not preserved in this way, and a function _might_ be part of the config.